src/main/resources/assets/css/style.css

```diff
@@ -13,6 +13,18 @@
 
 /* --- */
 
+.loading-icon {
+  width: 100px !important;
```

Comment:
Avoid !important if at all possible. 
Reorder your css to set a higher specificy then other stylesheets. 
If you have two `<link [...] />` elements the last one will have higher specificity